content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -194,3 +194,15 @@ This is available from the following web page: https://documentation.softwareag.
 Further to the CEL (Esper) deprecation notice in [release 10.5](/release-10-5-0/migration-10-5-0/) and the subsequent announcement of end of support in [release 10.7](/release-10-7-0/announcements-10-7-0/) it has been possible to continue using Esper in an unsupported mode. With effect from release 10.15 the CEL (Esper) functionality has been removed completely and cannot be used anymore.
 
 If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).
+
+### Cumulocity IoT DataHub
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - support for mixed types will be deprecated
```

Comment:
##### Breaking change in the offloading mechanism - DEPRECATION OF support for mixed types

(We deprecate with this release and will remove support in future versions)

Reply:
Done...

@BeateRixen  Can you pls merge the latest commit? Thanks!